site/_posts/2015-01-20-jekyll-meet-and-greet.markdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: news_item
+title: "Jekyll Meet & Greet at GitHub HQ"
+date: "2015-01-20 19:23:12 -0800"
+author: parkr
+categories: [meetup]
+---
+
+Hey! Our friends at GitHub have agreed to host a Jekyll meet & greet on
+**February 5, 2015 at 7pm**. The event will be hosted at [GitHub's
+Headquarters](https://www.google.com/maps/place/GitHub/@37.782084,-122.391403)
```

Comment:
I would go but there are some logistical problems:

1. There is no spot for me to park to display my my George Bush sticker.
2. I don't think I could get my Hummer H1 around the corner... corner parkers.